src/Umbraco.Web.UI.Client/src/less/components/umb-table.less

```diff
@@ -101,16 +101,21 @@ input.umb-table__input {
     }
 }
 
-.umb-table-body__icon {
+.umb-table-body__icon,
+.umb-table-body__icon[class^="icon-"],
+.umb-table-body__icon[class*=" icon-"] {
     margin: 0 auto;
-
     font-size: 22px;
+    line-height: 22px;
     color: @blueDark;
 }
 
-.umb-table-body__checkicon {
+.umb-table-body__checkicon,
+.umb-table-body__checkicon[class^="icon-"],
+.umb-table-body__checkicon[class*=" icon-"] {
     display: none;
     font-size: 16px;
+    line-height: 22px;
 }
```

Comment:
Fix issue with custom icon in listview list layout.
Because checkmark use a small font-size (16px) we also ensure that line-height match the largest of the font-sizes, so the row doesn't "jump" when selecting/unselecting the row.
The checkmark could use same font-size (22px), but maybe too big - I think it is fine with just 16px for checkmark icon.